lib/tasks/combined_tests.rake

```diff
@@ -5,7 +5,7 @@
 # require "bundler/gem_tasks"
 require "rake/testtask"
 
-Rake::Task['test:run'].clear
+Rake::Task["test:run"].clear
```

Comment:
I'm just curious: what was the difference here between using single and double quotes?  Was there a problem that got fixed, or is it more of a cosmetic thing?

Reply:
No difference.  Reverted to original